cms/src/test/resources/test-data/api-specifications/ApiSpecSyncFromApigeeJobIntegrationTest/oasV3_complete.html

```diff
@@ -4,7 +4,7 @@
         <span id="sticky-nav-header" class="nhsd-t-heading-xs nhsd-!t-margin-bottom-2">Page content</span>
```

Comment:
should either this or the other oasV3_complete file be changed to a symlink so we only have to update one? just a thought

Reply:
Another good point.

My worry is how well the symlinks would transfer between Linux, Windows and Mac.

Speaking of which, we should probably replace this one with a strongly slimmed-down version. I'll see if I can sneak it in in my current ticket.